GameOfLifeApp/Display.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLifeApp
+{
+    /// <summary>
+    /// Displays 2 boolean values array. 
+    /// </summary>
+    public class Display
+    {
+        /// <summary>
+        /// Insert appropriate element in the array.
```

Comment:
Check if summary is correct. Method does not do any inserts

Reply:
Changed.